README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/gap-system/gap.svg?branch=master)](https://travis-ci.org/gap-system/gap) [![Code Coverage](https://codecov.io/github/gap-system/gap/coverage.svg?branch=master&token=)](https://codecov.io/gh/gap-system/gap)
+[![Build Status](https://travis-ci.org/gap-system/gap.svg?branch=master)](https://travis-ci.org/gap-system/gap) [![Build Status](https://travis-ci.org/gap-system/gap-docker-master-testsuite.svg?branch=master)](https://travis-ci.org/gap-system/gap-docker-master-testsuite) [![Code Coverage](https://codecov.io/github/gap-system/gap/coverage.svg?branch=master&token=)](https://codecov.io/gh/gap-system/gap)
```

Comment:
I am unhappy that this makes two visually indistinguishable Travis badges appear. And worse, both have the same alt string "Build Status" -- so the only way to find out which is which is to check the URLs.

So the first change should be to use different "alt" strings, say "Build status" or "master branch status", versus e.g. "Docker build status".

Secondly, I'd strongly prefer if there also was an immediate visible clue as to which badge is is which. I don't know if the Travis badges allow that, but <https://shields.io> certainly does (and according to [this blog post](https://blog.travis-ci.com/2014-03-20-build-status-badges-support-svg/) they are the source of Travis badges anyway?):
* Default shield for gap-system/gap: [![master branch build status](https://img.shields.io/travis/gap-system/gap.svg)](https://travis-ci.org/gap-system/gap)
* Shield with custom label: [![master branch build status](https://img.shields.io/travis/gap-system/gap.svg?label=master%20branch)](https://travis-ci.org/gap-system/gap)

There are other options (e.g. to adjust colors, styling etc.), describe on their homepage.